templates/partials/social_media.html

```diff
@@ -0,0 +1,24 @@
+<div class="social-media-section">
+    <h3>{{ social_media_title }}</h3>
+
+    {% set link_icon_array = ["images/logos/social-media/icon_Linkedin.svg",
+    "images/logos/social-media/facebook-v1.svg",
```

Comment:
I don't think we have a facebook account

Reply:
Removed the icon